### ISSUE TITLE
Some campaigns cannot be deleted

### ISSUE THREAD

**Issue by phocean** [gophish/gophish#268]
Hello,

For some unknown reason, some campaign cannot be deleted.

I get such output on stdout:

```
127.0.0.1 - - [12/May/2016:16:59:44 +0200] "DELETE /api/campaigns/14?api_key=c6196ce435e69cfef2b117fbf991079f91099b384b21fe96229a8553e3dd263c HTTP/1.1" 404 73 "http://127.0.0.1:3333/" "Mozilla/5.0 (X11; Linux x86_64; rv:46.0) Gecko/20100101 Firefox/46.0"
 2016/05/12 17:00:08 campaign.go:174: record not found: template not found for campaign
 2016/05/12 17:00:08 api.go:95: record not found
```
When I display them, I also get errors:

```
127.0.0.1 - - [12/May/2016:17:01:05 +0200] "GET /campaigns HTTP/1.1" 200 10853 "http://127.0.0.1:3333/" "Mozilla/5.0 (X11; Linux x86_64; rv:46.0) Gecko/20100101 Firefox/46.0"
 2016/05/12 17:01:06 campaign.go:140: record not found
 2016/05/12 17:01:06 campaign.go:140: record not found
 2016/05/12 17:01:06 campaign.go:140: record not found
```

Let me know if I can help debugging.

**Comment by jordan-wright** [comment]
What version are you using?

**Comment by phocean** [comment]
It is the v0.1.2 binary.

**Comment by jordan-wright** [comment]
You might try building from source - I believe this was fixed in the upcoming release.

**Comment by jordan-wright** [comment]
After further review, this appears to have been fixed as part of #197. Let me know if building from source doesn't fix the issue!

Thanks!
